about/ATTRIBUTION.md

```diff
@@ -2,33 +2,33 @@ OpenSource Licenses for Terraform Provider for PowerStore
 =======================================================================
 
 Package: terraform-plugin-docs  
-Version: v0.13.0  
+Version: v0.18.0  
```

Comment:
is this expected?

Reply:
Yes, this was updated in the last release `1.1.2` but was missed in the ATTRIBUTIONS page.